plugins/ooni/main.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "OONI",
+    "name": "ooni",
```

Comment:
Correct. This is only an issue because the plugin was created by hand. This isn't a problem for plugins that are installed as NM creates the folder name based on the "name" field.